examples/server/server.cpp

```diff
@@ -671,6 +671,12 @@ struct server_context {
             model = nullptr;
         }
 
+	// Clear any sampling context
+	for (server_slot & slot : slots) {
+	    if (slot.ctx_sampling != nullptr)
+	        llama_sampling_free(slot.ctx_sampling);
+	}
```

Comment:
This LGTM, but would you mind fixing indentation? (Here we use 4 spaces)

Also would be nice to have brackets for the `if` (because we also using brackets for the `for` loop)